ansible/roles/bastion/tasks/k8s.yml

```diff
@@ -0,0 +1,18 @@
+---
+- tags:
+    - bastion_k8s
+  block:
+  # TODO: (jbirchler) Ask JimR if this should be deleted (it is deleted upstream)
+  # Note: EPEL must be enabled "somewhere". Currently
+  # EPEL comes from {own_repo_path}/{osrelease}/epel
+  # Repo needs to be enabled in repos_template.j2
+  - name: Install Python2 OpenShift Library (and dependencies)
+    yum:
+      state: present
+      name:
+      - https://gpte-public.s3.amazonaws.com/python2-pip-8.1.2-9.el7.noarch.rpm
+      - python2-openshift
+  - name: Install virtualenv
```

Comment:
We depend on virtualenv to be installed.

Reply:
If they don't do it, we do.

Reply:
This is Python 2 -- which has been retired. We  can bring up to Jim (i.e. Python 2 vs Python 3)

Reply:
In any case, this file should removed. It was deleted upstream.